### PR TITLE
Port TG Backpacks, Duffelbags and Satchels - can't stop addicted to the shindig

### PR DESCRIPTION
**Pull request by Alecksohs**
## What Does This PR Do

Ports the backpacks duffelbags and satchels from TG

## Why It's Good For The Game

better

## Images of changes

image

Okay now just imagine the rest heres the spritesheet this took 40 minutes to setup and I'm not doing it again for other backpacks lmfao

image
image
image
image
image
image
image
image
image

## Testing

I loaded it ingame and tested every sprite. I just didn't care to make a gallery of them working.

## Changelog
:cl:
Imageadd: Ported TG Backpacks
/:cl:

### CODE REVIEW

**Comment by Alecksohs** [comment]
> Your PR has been put on hold as your GitHub account is not linked to a Paradise forums account.
> 
> Please link your account at https://www.paradisestation.org/forum/custom/link_github_account.php
> 
> _Headcoders may ignore this in specific circumstances (Minor fix PR, trusted contributor from another codebase, anything at their discretion)_

ugly

**Comment by Henri215** [comment]
"imageadd:" instead of "tweak:" over your changelog

**Comment by DT2399** [comment]
Nicee!

**Comment by Burzah** [comment]
I see that the testing field is empty. Were all the sprites that are included get examined in game?

**Comment by Alecksohs** [comment]
> I see that the testing field is empty. Were all the sprites that are included get examined in game?

idk what you mean its not empty smh my head

**Comment by ATP-Engineer** [comment]
Can you do the same for holder bags such as viro, chem, and botany.

**Comment by DGamerL** [comment]
Science uses a different icon than the `!` here, could you update it to our job icons?

**Comment by Christasmurf** [comment]
While I'm here I should also suggest swapping the red cross on the syndiemed duffel to either white or green, we don't use red crosses for medical symbols as it's a war crime

**Comment by Henri215** [comment]
image
Any idea why the diff bot is showing everything inside the file?

**Comment by S34NW** [comment]
> image Any idea why the diff bot is showing everything inside the file?

image
these two files are top-level copies likely left in error

**Review by S34NW — CHANGES_REQUESTED** [review]
Please properly address where the files are and make sure the changes are working, things are in the wrong folders as pointed out above.

**Review by S34NW — CHANGES_REQUESTED** [review]
PR stale, please address reviews within a week or the PR will be closed

**Comment by Alecksohs** [comment]
Will be updating, had been burntout when it was made.

File: back.dmi
Comment (S34NW) [inline back.dmi:1]: You still add this file to the root of the codebase structure, please address this

**Review by S34NW — CHANGES_REQUESTED** [review]

File: back.dmi
Comment (Alecksohs) [reply back.dmi:1]: For now, I'm going to be closing this as I am busy with other projects, I may reopen it at another date.

**Comment by Alecksohs** [comment]
For now, I'm going to be closing this as I am busy with other projects, I may reopen it at another date.

**Pull request closed without merging**